VueJS/App.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <div id="app">
+    <Header></Header>
+    <AddTodo v-on:addTodo="addTodo"></AddTodo>
```

Comment:
```suggestion
    <AddTodo v-on:add-todo="addTodo"></AddTodo>
```
I recently learned that there is a convention to use kebab case for event names: https://vuejs.org/v2/guide/components-custom-events.html#Event-Names